sdk/keyvault/azure-security-keyvault-certificates/test/samples/certificate-get-certificates/CMakeLists.txt

```
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
+
+project (certificate-get-certificates LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+add_executable (
+  certificate-get-certificates
+  certificate_get_certificates.cpp
+)
+
+target_link_libraries(certificate-get-certificates PRIVATE azure-security-keyvault-certificates azure-identity )
```

nit: remove whitespace
```suggestion
target_link_libraries(certificate-get-certificates PRIVATE azure-security-keyvault-certificates azure-identity)
```

---

Not related to this PR, but sure, let's take it